repo/blob/s3/s3_storage_test.go

```diff
@@ -402,18 +457,22 @@ func createMinioSessionToken(t *testutil.RetriableT, minioEndpoint, kopiaUserNam
 	return *result.Credentials.AccessKeyId, *result.Credentials.SecretAccessKey, *result.Credentials.SessionToken
 }
 
-func cleanupOldData(ctx context.Context, t *testutil.RetriableT, options *Options) {
+func cleanupOldData(ctx context.Context, tb testing.TB, options *Options, cleanupAge time.Duration) {
```

Comment:
question / nit: Should the parameter name `tb` remain the same as before => `t`

Reply:
the linter flagged it, so I just fixed it.

Reply:
I see :(